packages/block-editor/src/components/block-list/use-block-props/index.js

```diff
@@ -134,7 +134,7 @@ export function useBlockProps( props = {}, { __unstableIsHtml } = {} ) {
 		ref: mergedRefs,
 		id: `block-${ clientId }${ htmlSuffix }`,
 		tabIndex: 0,
-		role: 'group',
+		role: undefined,
```

Comment:
Why not just remove the property entirely (I mean dropping this line), I guess same thing for the tests.

Reply:
@youknowriad I didn't remove it because rich-text defines a default role of textbox. Specifying undefined here overrides the role in passed props.

Either way, the tests would have needed changing. Seems this effects quite a few of them.

Reply:
so we don't want to retain the `textbox` role for RichTexts? Why some RichText should have it and others not?

Reply:
I don't really see the need for it. contentEditable seems to work fine unless others see a specific reason we should retain this role. It will need a fair amount of testing to be sure.

Reply:
Fair enough, I think this was originally proposed by @afercia. We'd have to dig up the history to check the reasoning there.